Pipfile

```diff
@@ -17,6 +17,10 @@ mysqlclient = "==1.3.13"
 mimesis = "==2.1.0"
 bleach = "==3.0.2"
 python-dateutil = "*"
+unidecode = "*"
+celery = "==4.1.0"
+arxiv-auth = "*"
+"c6a1273" = {path = "./core"}
```

Comment:
What is this syntax? Is the hash from the PIpfile.lock? Very curious to learn more.

Reply:
Relative path; to install the package in`` ./core``